openedx/core/djangoapps/course_live/tests/test_tab.py

```diff
@@ -0,0 +1,91 @@
+"""
+Tests for  course live tab.
+"""
+import itertools
+from unittest.mock import Mock, patch
+
+import ddt
+from django.test import RequestFactory
+from edx_toggles.toggles.testutils import override_waffle_flag
+from lti_consumer.models import CourseAllowPIISharingInLTIFlag, LtiConfiguration
+
+from lms.djangoapps.courseware.tests.test_tabs import TabTestCase
+from openedx.core.djangoapps.course_live.config.waffle import ENABLE_COURSE_LIVE
+from openedx.core.djangoapps.course_live.models import CourseLiveConfiguration
+from openedx.core.djangoapps.course_live.tab import CourseLiveTab
+
+
+@ddt.ddt
+class CourseLiveTabTestCase(TabTestCase):
+    """Test cases for LTI CourseLive Tab."""
+
+    def setUp(self):
+        super().setUp()
+        self.course_live_config = CourseLiveConfiguration.objects.create(
+            course_key=self.course.id,
+            enabled=False,
```

Comment:
Should this be `false`?

Reply:
Yes, these are updated in test cases as required.